### PR TITLE
Add non-english subsection to README.md

### PR DESCRIPTION
**Pull request by Tronde**
As discussed during today's Ansible Contribution Summit I'd like to propose a subsection for non-english content.  This PR shows a new subsection in 'Blog posts and opinions'.

There is just the beginning of a new list. The language of the content is mentioned in the entry. In case there will be multiple different languages in the future the list could be sorted or new subsections for different languages could be introduced.

This example could be used for other sections like videos, tools, etc. too.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,6 +103,10 @@
 - [Testing Ansible Roles Against Windows with Test-Kitchen](https://hodgkins.io/testing-ansible-roles-windows-test-kitchen) - Using Test-Kitchen with Ansible to apply playbooks to Windows machines and test them with [Pester](https://github.com/pester/Pester/).
 - [Ansible Best Practices by AndiDog](https://andidog.de/blog/2017-04-24-ansible-best-practices) - Practices covering many aspects of an Ansible setup, including hints to support different environments (testing, staging, production).
 
+### Other languages than English
+
+- [Ansible auf My-IT-Brain (German)](https://www.my-it-brain.de/wordpress/category/ansible/) - Articles about different topics regarding Ansbile, e.g. modules, roles, use cases, etc.
```
Comment (gundalow) [inline README.md:108]: This is a great idea, thank you.

I expect that the "Blogs" section will have quite a few non-English articles added, I wonder if they should just be grouped alphabetically by language, ie

```suggestion
### German

- [Ansible auf My-IT-Brain](https://www.my-it-brain.de/wordpress/category/ansible/) - Articles about different topics regarding Ansbile, e.g. modules, roles, use cases, etc.
```

**Review by gundalow — APPROVED** [review]
Looks great, thank you.

**Review by KeyboardInterrupt — APPROVED** [review]
I like this addition! :+1:
Looks good to me.

**Comment by KeyboardInterrupt** [comment]
Actually I was so excited for the Idea of non English resources, I totally forgot to comment on the Content itself ;)
I might have prematurely merged it!
I think it would be best to NOT point to a Blog's Topic, I would rather like to point to specific Blog Article(s).
@Tronde Is there a specific Article you would like to add to this List?

**Comment by Tronde** [comment]
> Actually I was so excited for the Idea of non English resources, I totally forgot to comment on the Content itself ;) I might have prematurely merged it! I think it would be best to NOT point to a Blog's Topic, I would rather like to point to specific Blog Article(s). @Tronde Is there a specific Article you would like to add to this List?

Hi @KeyboardInterrupt, It seems my email answer didn't make it into this conversation. Sure, I could add some specific articles to the list. Would you like to get one PR per article? Or should I put a bunch of them in one PR?

Please give me some time to curate them.

**Comment by KeyboardInterrupt** [comment]
I will get back to you on that some time later this Week, I am rather busy right now with other obligations.

**Pull request merged**